src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double sideB;
+
+    public Rectangle(Colors color, double side, double sideB, double height) {
+        super(color, side);
+        this.sideB = sideB;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Isosceles Trapezoid, area: "
+                + getArea() + "sq. units, side "
+                + getSide() + " units, sideB " + sideB
+                + " units, color: " + getColor().name());
+    }
+
+    @Override
+    public double getArea() {
```

Comment:
The formula for calculating the area of a rectangle is incorrect. It should be the product of two different sides. Make sure to use both `side` and `sideB` to calculate the area.